pkg/cmd/server/origin/master.go

```diff
@@ -889,6 +897,13 @@ func (c *MasterConfig) RunDeploymentImageChangeTriggerController() {
 	controller.Run()
 }
 
+// SDN controller runs openshift-sdn if the said network plugin is provided
+func (c *MasterConfig) RunSDNController() {
+	if c.Options.NetworkConfig.NetworkPluginName == osdn.NetworkPluginName() {
```

Comment:
Reading down, it looks like excluding the NetworkPluginName from the master config is that flag.